wca-regulations.md

```diff
@@ -169,6 +169,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 4d2) Pyraminx is scrambled starting with the yellow face (if not possible, then the lightest face) on the bottom and the green face (if not possible, then the darkest adjacent face) on the front.
     - 4d3) Square-1 is scrambled starting with the darker color on front (out of the 2 possible scrambling orientations).
     - 4d4) Clock is scrambled starting with either side in front and 12 o'clock pointing up.
+        - 4d4a) After scrambling the puzzle, the scrambler must orient all pins up, from the position the puzzle is held.
```

Comment:
12 o'clock is already up, and the notation is consistent with the clock faces being on F and B. I'd write it like this:

```suggestion
        - 4d4a) After scrambling the puzzle, the scrambler must push all pins to the front (sometimes informally described as "pushed up"), from the position the puzzle is held.
```